explainer.md

```diff
@@ -91,19 +103,15 @@ async function OnVRAvailable() {
   if (exclusiveMode) {
     var enterVrBtn = document.createElement("button");
     enterVrBtn.innerHTML = "Enter VR";
-    enterVrBtn.addEventListener("click", BeginVRSession());
+    enterVrBtn.addEventListener("click", BeginVRSession);
     document.body.appendChild(enterVrBtn);
   }
 }
 ```
 
 ### Beginning a VR session
 
-Clicking that button will attempt to initiate a [`VRSession`](https://w3c.github.io/webvr/#interface-vrsession), which manages input and output for the display. When creating a session with `VRDevice.requestSession` the capabilities that the returned session must have are passed in via a dictionary, exactly like the `supportsSession` call. If `supportsSession` returned true for a given dictionary then calling `requestSession` with the same dictionary values should be reasonably expected to succeed, barring external factors (such as `requestSession` not being called in a user gesture for an exclusive session or another page currently having an exclusive session for the same device.)
-
-The page may also want to create a session that doesn't need exclusive access to the device for tracking purposes. Since the `VRSession` is also what provides access to the device's position and orientation data requesting a non-exclusive session enables what's referred to as "Magic Window" use cases, where the scene is rendered on the page normally but is responsive to device movement. This is especially useful for mobile devices, where moving the device can be used to look around a scene. Devices with Tango tracking capabilities may also expose 6DoF tracking this way, even when the device itself is not capable of stereo presentation.
-
-Requesting a new type of session will end any previously active ones.
+Clicking that button will attempt to acquire a `VRSession` by callling `VRDisplay.requestSession`. This returns a promise that resolves to a `VRSession` upon success. When requesting a session, the capabilities that the returned session must have are passed in via a dictionary, exactly like the `supportsSession` call. If `supportsSession` returned true for a given dictionary, then calling `requestSession` with the same dictionary values should be reasonably expected to succeed, barring external factors (such as `requestSession` not being called in a user gesture for an exclusive session or another application currently having an exclusive session for the same VR hardware device.)
```

Comment:
"another application currently having an exclusive session for the same VR hardware device" isn't true for all VR platforms.  :)  Can we reword this to be a bit clearer by perhaps removing this reason and just saying that it's up to the underlying platform to decide if it can honor the request?